build.gradle

```diff
@@ -95,10 +95,12 @@ jacocoTestReport {
 
 //NOTE: we ignore contracts for now
 compileJava {
-  options.compilerArgs = ['-proc:none', '-Xlint:all','-Werror', '-Xdiags:verbose']
+  // removing -Werror temporarily
```

Comment:
Could you put `-Werror` back?  Is there a warning you couldn't resolve?

Reply:
I put -Werr back in, it works. Thanks!